packages/opentelemetry-tracing/src/config.ts

```diff
@@ -47,6 +47,4 @@ export const DEFAULT_CONFIG = {
     numberOfLinksPerSpan: DEFAULT_MAX_LINKS_PER_SPAN,
     numberOfEventsPerSpan: DEFAULT_MAX_EVENTS_PER_SPAN,
   },
-  // @todo add support for plugins
```

Comment:
This has been implemented for `NodeTracer` (https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-node/src/config.ts#L23-L42), this is not applicable for `BasicTracer` config.